codingTest_kjy/셔울에서김서방찾기.js

```diff
@@ -0,0 +1,4 @@
+function solution(seoul) {
+    let answer = seoul.IndexOf("Kim")
```

Comment:
```suggestion
    const answer = seoul.IndexOf("Kim")
```
answer는 처음 할당할때 이후에 값이 변경되지 않기 때문에 const로 선언해주시면 될 것 같습니다.

Reply:
확인했습니다!